index.d.ts

```diff
@@ -1,3 +1,19 @@
+// TypeScript Version: 3.0
+
+export interface HeadersDefaults {
+  common: Record<string, string>;
+  delete: Record<string, string>;
+  get: Record<string, string>;
+  head: Record<string, string>;
+  post: Record<string, string>;
+  put: Record<string, string>;
+  patch: Record<string, string>;
+  options?: Record<string, string>;
+  purge?: Record<string, string>;
+  link?: Record<string, string>;
+  unlink?: Record<string, string>;
+}
```

Comment:
Wouldn't it be better to create a `Headers` alias instead of repeating this record over and over again? Also, wouldn't it be more helpful to allow `Record<string, string | number | boolean>` for headers?